experiments/vereya/test_observation.py

```diff
@@ -73,6 +73,13 @@ def test_observation_from_ray(self):
         dist = self.getDist()
         self.assertGreater(dist, 0)
 
+    def test_observation_from_chat(self):
+        self.mc.sendCommand("chat get wooden_axe")
+        time.sleep(0.5)
+        self.rob.observeProcCached()
```

Comment:
You'll have to use rob.waitNotNoneObserve('getChat') because chat message might disappear from observations

Reply:
@noskill 
So something like that?
chat = self.rob.waitNotNoneObserve('getChat')

If yes, then i'm getting None here.
Full code
   ```
 def test_observation_from_chat(self):
        self.mc.sendCommand("chat get wooden_axe")
        time.sleep(0.5)
        self.rob.observeProcCached()
        chat = self.rob.waitNotNoneObserve('getChat')
```
chat vairiable is None here.